packages/app/cypress.config.ts

```diff
@@ -38,6 +38,7 @@ export default defineConfig({
     pluginsFile: 'cypress/e2e/plugins/index.ts',
     supportFile: 'cypress/e2e/support/e2eSupport.ts',
     async setupNodeEvents (on, config) {
+      // process.env.DEBUG = '*'
```

Comment:
should we remove this?

Reply:
I think it might be useful to keep here so it's clear that you can uncomment or change if you want to debug things